component/common/src/main/java/org/wso2/carbon/identity/smsotp/common/dto/FailureReasonDTO.java

```diff
@@ -46,6 +46,14 @@ public FailureReasonDTO(Constants.ErrorMessage error, String data) {
                 : error.getDescription();
     }
 
+    public FailureReasonDTO(Constants.ErrorMessage error, String data, int remainingFailedAttempts) {
```

Comment:
Better to add a method comment since this a public method

Reply:
Resolved with 69b280575c654b56917dae1fdf53ea71618ef29a